src/assets/scss/global/typography.scss

```diff
@@ -1,13 +1,5 @@
 @import "settings";
 
-h1.huge {
-	@include huge-headline();
-}
-
-.small-text {
-	@include small-text();
-}
-
 .featured-text {
```

Comment:
These remaining 3 classes are still in use on Homepage[Contentful] components. I'm not sure which are still in play. Probably best to wait for CORE-51 to remove.

Reply:
Oh interesting...

Reply:
@ryan-ludwig can you make a note of this in CORE-51 so we don't forget to remove these classes when that work is done